htmd/charge/charge.py

```diff
@@ -69,6 +70,92 @@ def fitGasteigerCharges(mol):
     return mol
 
 
+def fitChargesWithAntechamber(mol, type='gas', molCharge=None):
```

Comment:
Are you using this somewhere? I could not find the reference.

Reply:
Sorry, didn't see the comment above. Could you just add it as a BCC option for now? I need this to actually test my molecule dataset against parameterize

Reply:
AM1-BCC will be another PR. I also need implement symmetrization for them